guides/source/association_basics.md

```diff
@@ -783,7 +801,8 @@ The following sections give the details of each type of association, including t
 
 ### `belongs_to` Association Reference
 
-The `belongs_to` association creates a one-to-one match with another model. In database terms, this association says that this class contains the foreign key. If the other class contains the foreign key, then you should use `has_one` instead.
+In database terms, the `belongs_to` association says that this class contains the foreign key. This can be used to set up one-to-one, one-to-many or a many-to-many relation, depending on setup.
```

Comment:
That's not correct either. A foreign key is a way the database can guarantee referential integrity. It is not required for the belongs_to method to work though. Some databases don't even have foreign keys.
Also: many_to_many relations require a separate ```JoinTable```, so it does not belong in this section.

I would suggest something like:

In database terms, the `belongs_to` association says that this model's table contains a column which represents a reference to another table. This can be used to set up one-to-one or one-to-many relations, depending on the setup.

Reply:
You're right, the term is ambiguous and we do not need to go into strict database theory here.